lang/src/main/pig/partiql.ion

```diff
@@ -142,14 +142,16 @@ may then be further optimized by selecting better implementations of each operat
             (bag_op op::bag_op_type quantifier::set_quantifier operands::(* expr 2))
 
             // GPML graph pattern match:  <expr> MATCH <gpml_pattern>
-            (graph_match expr::expr gpml_pattern::gpml_pattern)
+            experimental::(graph_match expr::expr gpml_pattern::gpml_pattern)
```

Comment:
The intent of this annotation is that people developing w.r.t. partiql-kotlin-lang as a library are informed that the (PIG-generated) AST node GraphMatch is experimental. 

However, the informing is done by showing the `org.partiql.pig.runtime.Experimental` annotation, which comes from PIG.  It feels like that should be an annotation coming from PartiQL, such as `org.partiql.annotations.PartiQLExperimental`. That is, we should shield the user from the PIG implementation detail.  
I realize that informing PIG of a specific PartiQL-defined annotation we want to use on the nodes PIG generates would require mutual dependency between PIG and PartiQL, which we can't/don't want to introduce.

I also realize that this might not be that visible to the user, since errors and warnings only show the message specified in the annotation. (Though, I am not certain the class name is opaque in all contexts.)  

So, with the opaqueness, I might be talked into leaving this as is, but there is a more serious problem that will transpire in due time: we really need an annotation per feature (in this PR, one for GPML and another for the Window feature), since these features will graduate on different timelines (see https://kotlinlang.org/docs/opt-in-requirements.html#opt-in-requirements-for-pre-stable-apis).  I don't see how this can be done without communicating these multiple experimental annotations to PIG -- running into the mutual dependency obstacle above. 

So, unless someone has a clever idea how to deal with this, I am inclined to advocate **against** making PIG put `RequiresOptIn` annotations on PIG-generated code.